red/red.js

```diff
@@ -36,6 +38,7 @@ var RED = {
         userSettings.version = this.version();
         settings.init(userSettings);
         server.init(httpServer,settings);
+        memoryMetricsReporter.init(); //TODO pass in interval settings
```

Comment:
I'm considering whether making the memory manager initialization controllable via settings. I could add that feature straight away, therefore it wouldn't get initialized by default?...

Secondly, the memory metrics reporting frequency interval could also be configured via the settings, that might be something we could also want, even in the first commit of this feature?